kolibri/utils/version.py

```diff
@@ -64,13 +70,16 @@ def derive_version_from_git_tag():
         subprocess.CalledProcessError,  # not a git repo
         OSError  # git executable doesn't exist
     ):
-        return None
+        pass
+    else:
+        version_string = parse_git_tag_version_string(git_describe_string)
 
-    return parse_git_tag_version_string(git_describe_string)
+    os.chdir(current_dir)
+    return version_string
 
 
-def derive_version_from_version_file():
-    string = pkgutil.get_data('kolibri', 'VERSION').decode('utf-8')
+def derive_version_from_version_file(package):
+    string = pkgutil.get_data(package, 'VERSION').decode('utf-8')
```

Comment:
I don't quite understand - is there an action item here that means we shouldn't merge #1017?